### PR TITLE
Generate lib/dist files from npm install

### PR DESCRIPTION
**Pull request by derektli**
These changes are genearted from npm install. 
@notandrewkaye  @emilycw  Can you take a look to see if any of these change are unexpected?

### CODE REVIEW

File: lib/datepicker.js [hunk 1]
```diff
@@ -219,7 +220,8 @@ var DatePicker = _react2.default.createClass({
       timePickerButton: this.props.timeDisabled ? false : this.props.timePickerButton,
       onToggle: this.handleToggleTime,
       showTimePicker: this.state.showTimePicker,
-      onRemoveTime: this.handleRemoveTime });
+      onRemoveTime: this.handleRemoveTime,
+      fixedHeight: this.props.fixedHeight });
```
Comment (derektli) [inline lib/datepicker.js:224]: This looks weird to me.
I did not add fixedHeight into our code.

File: lib/month.js [hunk 2]
```diff
@@ -43,7 +44,7 @@ var Month = _react2.default.createClass({
     return [0, 1, 2, 3, 4, 5].map(function (offset) {
       return startOfMonth.clone().add(offset, 'weeks');
     }).filter(function (startOfWeek) {
-      return _this.isWeekInMonth(startOfWeek);
+      return _this.props.fixedHeight || _this.isWeekInMonth(startOfWeek);
```
Comment (derektli) [inline lib/month.js:47]: so is this.

File: lib/year_dropdown_options.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ var YearDropdownOptions = _react2.default.createClass({
         selectedYear === year ? _react2.default.createElement(
           'span',
           { className: 'react-datepicker__year-option--selected' },
-          '✓'
+          '\u2713'
```
Comment (derektli) [inline lib/year_dropdown_options.js:46]: ?

**Comment by derektli** [comment]
I have a feeling that we might have updated some `lib/*.js` instead of generating them in previous PR. 😱

File: lib/calendar.js [hunk 4]
```diff
@@ -74,17 +75,23 @@ var Calendar = _react2.default.createClass({
...
+    var _props = this.props,
+        selected = _props.selected,
+        openToDate = _props.openToDate;
 
     var minDate = (0, _date_utils.getEffectiveMinDate)(this.props);
     var maxDate = (0, _date_utils.getEffectiveMaxDate)(this.props);
     var current = (0, _moment2.default)();
     if (selected) {
       return selected;
+    } else if (minDate && maxDate && openToDate && openToDate.isBetween(minDate, maxDate)) {
```
Comment (derektli) [inline lib/calendar.js:87]: ?

File: lib/datepicker.js (hunk 1 above)
Comment (derektli) [reply lib/datepicker.js:224]: @emilycw It seems like you added this last year but it's never generated into lib? https://github.com/flexport/react-datepicker/commit/a42c8503bd1d4e2f8d212e86ee587aed64565ff3

File: lib/month.js (hunk 2 above)
Comment (derektli) [reply lib/month.js:47]: @emilycw https://github.com/flexport/react-datepicker/commit/a42c8503bd1d4e2f8d212e86ee587aed64565ff3

File: lib/calendar.js (hunk 4 above)
Comment (derektli) [reply lib/calendar.js:87]: @emilycw a42c8503bd1d4e2f8d212e86ee587aed64565ff3

**Comment by bdj** [comment]
I agree it looks like we made changes without rebuilding, so we should test that everything from the last few commits to the library works the way we expect. I also wonder if we want to pull in upstream changes at some point, probably not this PR.

File: lib/calendar.js (hunk 4 above)
Comment (aubreycw) [reply lib/calendar.js:87]: Yeah, this (and the other comments) also looks to me like I just didn't update the lib after merging the changes into our fork :(

**Comment by derektli** [comment]
@notandrewkaye  Can you review this change especially the part that is generated from previous changes?

**Review by andrew-dash — APPROVED** [review]
I agree with Blake -  the `fixedHeight` prop isn't used in our app so I don't think that will be a problem and the new min / max conditionals _seem_ like they won't cause issues.

**Comment by derektli** [comment]
copied over and tested in core left.

**Pull request merged**